CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Keycloak Community
+
+The Keycloak Node.js adapter is an Open Source adapter solution to protect modern Node.js Applications and Services.
+
+## Contributing to the Node.js adapter
+
+The Node.js adapter is an Open Source community-driven project and we welcome contributions as well as feedback from the community.
```

Comment:
Why do we reference the Node,js adapter here ?

Reply:
Because I made a mistake :)